test/help.ts

```diff
@@ -10,31 +10,40 @@ import microTemplateServiceLib = require("../lib/common/services/micro-templatin
 import dynamicHelpServiceLib = require("../lib/common/services/dynamic-help-service");
 import dynamicHelpProviderLib = require("../lib/dynamic-help-provider");
 import htmlHelpServiceLib = require("../lib/common/services/html-help-service");
-let opts = require("../lib/common/options");
+
+import optionsLib = require("../lib/options");
+import hostInfoLib = require("../lib/common/host-info");
 
 let assert = require("chai").assert;
 
-let createTestInjector = (options?: { isProjectTypeResult: boolean; isPlatformResult: boolean }): IInjector => {
-	opts.help = true;
+let createTestInjector = (opts?: { isProjectTypeResult: boolean; isPlatformResult: boolean }): IInjector => {
 	let injector = new yok.Yok();
 	let logger = new stubs.LoggerStub();
-	injector.register("logger", logger);
+	injector.register("hostInfo", hostInfoLib.HostInfo);
+	injector.register("staticConfig", {
+		helpTextPath: path.join(__dirname, "../resources/help.txt")
+	});
 	injector.register("errors", stubs.ErrorsStub);
-	options = options || { isPlatformResult: true, isProjectTypeResult: true };
+	injector.register("options", optionsLib.Options);
+	let options = injector.resolve("options");
```

Comment:
This is a stylistic option, but consider naming variables coming from the injector with leading $, so that find in files can easily find them.